lib/lightning/projects/project.ex

```diff
@@ -67,8 +65,6 @@ defmodule Lightning.Projects.Project do
     |> validate_length(:description, max: 240)
     |> validate_required([:name])
     |> validate_format(:name, ~r/^[a-z\-\d]+$/)
-    |> validate_inclusion(:history_retention_period, @retention_periods)
-    |> validate_inclusion(:dataclip_retention_period, @retention_periods)
```

Comment:
I feel this should still be validated no? We're extending the validation via a hook, but these constraints are still valid here right?